app/src/components/keyCloakServiceClientMgr.js

```diff
@@ -169,17 +169,23 @@ class KeyCloakServiceClientManager {
     }
   }
 
-  async fetchAllClients() {
-    log.info('KeyCloakServiceClientManager.fetchAllClients');
+  async fetchAllClients(kcEnv) {
+
+    log.info('KeyCloakServiceClientManager.fetchAllClients in ' + kcEnv);
 
     //get all service clients
-    // NOTE: this seems to return all the service clinets but shows a ERR 400 in the console for all the methods in the reamlAdminSvc class
-    const clients = await this.svc.getClients();
+    let clients = await this.svc.getClients();
+
+    // remove those that dont match '*_SERVICE_CLIENT';
+    const regex = '.*_SERVICE_CLIENT$';
+    clients = clients.filter(cl => cl.clientId.match(regex));
```

Comment:
Since we aren't really using this regex anywhere else, you can just combine this into the following line
`clients.filter(cl => cl.clientId.match(/.*_SERVICE_CLIENT$/))`